src/components/calcite-filter/calcite-filter.tsx

```diff
@@ -58,17 +58,18 @@ export class CalciteFilter {
 
   filter = debounce((value: string): void => {
     const regex = new RegExp(value, "ig");
-    const find = (input, regex) => {
+
+    const find = (input: object, RE: RegExp) => {
       let found = false;
-      forIn(input, (value) => {
-        if (typeof value === "function") {
+      forIn(input, (val) => {
```

Comment:
[Nitpick] Why the rename?

Reply:
Because of this tslint rule: https://palantir.github.io/tslint/rules/no-shadowed-variable/